09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,70 @@
+main {
+  text-align:center;
+  color:white;
+  width: 100vw;
+  height: 100vh;
+  background: url(../img/1.jpg);
+  background-size: cover;
+}
+.opacity {
+  background-color: rgba(0, 0, 0, 0.2);
+  width:100%;
+  height: 65vh;
+  padding-top:35vh;
```

Comment:
good job using the opacity class here! a similar tactic needs to be employed for the service page boxes.